src/components/AppointmentForm/AppointmentForm.jsx

```diff
@@ -51,15 +51,15 @@ const AppointmentForm = () => {
         <input
           type="text"
           placeholder="Full Name"
-          value={fullName}
-          onChange={(e) => setFullName(e.target.value)}
+          value={authData.fullName}
+          // onChange={(e) => setFullName(e.target.value)}
```

Comment:
Please, remove commented code at least will be used in a next commit, define in a comment //To-Do in this commit.